### ISSUE TITLE
Stops running after about a minute

### ISSUE THREAD

**Issue by casperversteeg** [xybu/onedrived-dev#118]
Whenever I try to start onedrived, I have to point it to the ngrok binary again, but whenever I do, it will run for about a minute, as `onedrived status` returns:
`onedrived -- pid: 18611, status: sleeping, uptime: 0m, %cpu: 0.0, %mem: 0.7`
which it will keep returning for about a minute, and then it will say
`onedrived -- not running`
If I try to start onedrived without the `NGROK=~/path/ngrok` bit, it will fail with exit code 1, and the debug shuts down with
`CRITICAL: MainThread: Error initializing webhook: Did not find ngrok executable "ngrok"`
So basically, onedrived is not working at all, even though it used to just fine. Rebooting my machine sometimes fixes it, sometimes doesn't. How can I fix this without rebooting my machine every time?

**Comment by derrix060** [comment]
@casperversteeg have you already synced all your files? If that's the case, the framework will be sleeping until it detects some changes in your local repository, or when a timeout set on your configuration is reached.

**Comment by casperversteeg** [comment]
No not all files are synced. There are local files on my machine that did not get uploaded to OneDrive.
The issue tends to present itself whenever my machine goes to sleep without explicitly shutting down onedrived, or loses network connection.
